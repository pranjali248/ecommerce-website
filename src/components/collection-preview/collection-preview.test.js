import React from "react";
import { shallow } from "enzyme";
import CollectionPreview from "./collection-preview.component";

describe("Testing collection preview component", () => {
  it("Shallow snapshot testing collection preview component", () => {
    expect(shallow(<CollectionPreview items={[]} />)).toMatchSnapshot();
  });
});
