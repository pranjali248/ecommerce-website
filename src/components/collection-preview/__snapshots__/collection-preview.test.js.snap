// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing collection preview component Shallow snapshot testing collection preview component 1`] = `ShallowWrapper {}`;
